### ISSUE TITLE
Configurating while it's connected to the network.

### ISSUE THREAD

**Issue by augisbud** [tzapu/WiFiManager#693]
Hello, I want to be able to configure wifi and other settings while it's connected to my network. Any help is appreciated.

**Comment by augisbud** [comment]
#256 Solved.

**Comment by tablatronix** [comment]
Well the real solution is the development branch ondemandwebportal example

**Comment by augisbud** [comment]
Well from what I can see it just creates an extra AP with is not what I want. My main goal is to be able to enter the config portal from the network the esp is connected without creating extra AP's.

**Comment by augisbud** [comment]
Is it possible to start the config portal without an extra wifi?

**Comment by tablatronix** [comment]
`ondemandwebportal`
startWebPortal

**Comment by augisbud** [comment]
What do you mean exactly?

**Comment by augisbud** [comment]
Thank you, I'm looking into it

**Comment by augisbud** [comment]
Ok let me get this straight, I need to have the web interface always running alongside a dsc alarm code, i have them both, I'm gonna merge them, but how do I make the web interface always running alongside other code and have mqtt data written to spiffs?

**Comment by RobbesU** [comment]
See [this](https://www.youtube.com/watch?v=saAv7YOiAyM) video how to include custom parameters with the WiFiManager. Use (currently) with the development branch. This will run happily next to your logging code (especially on ESP32 where the wifi runs on core 0 and main code loop on core 1.  Works for me.
But shouldn't you only have to run wifi setup at startup ?

**Comment by augisbud** [comment]
I have added my items to wifimanager, thank you for the reminder that esp32 is dual core will try on it

**Comment by tablatronix** [comment]
Yeah that is what that example does, web portal only no ap or captive portal
